.travis.yml

```diff
@@ -14,6 +13,7 @@ before_install:
   - sh -e /etc/init.d/xvfb start
 deploy:
   provider: npm
+  script: ./deploy.sh
```

Comment:
This doesn't set it as a deploy target. See https://docs.travis-ci.com/user/deployment#Deploying-to-Multiple-Providers